partiql-tests-data/eval/query/undefined-variable-behavior.ion

```diff
@@ -0,0 +1,71 @@
+undefined_variable_behavior::[
+  // ported tests run using UndefinedVariableBehavior.MISSING option.
+  // https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/src/org/partiql/lang/eval/CompileOptions.kt#L27-L36
+  // EvalModeCoerce will return missing for undefined variable references. EvalModeError will give an error for
```

Comment:
This set of tests was testing the Kotlin implementation using the `UndefinedVariableBehavior` [option](https://github.com/partiql/partiql-lang-kotlin/blob/main/lang/src/org/partiql/lang/eval/CompileOptions.kt#L27-L36). Undefined variable behavior wasn't explicitly defined in the spec (other than for missing attributes), but I believe `EvalModeCoerce` corresponds to `UndefinedVariableBehavior.MISSING` (undefined variables return `MISSING`) and `EvalModeError` corresponds to `UndefinedVariableBehavior.ERROR` (undefined variables return an `ERROR`).